src/inc/corinfo.h

```diff
@@ -2111,13 +2121,6 @@ class ICorStaticInfo
             BOOL                        *pfIsOpenDelegate /* is the delegate open */
             ) = 0;
 
-    // Determines whether the delegate creation obeys security transparency rules
```

Comment:
While you are on it you can also delete `getAddrModuleDomainID` ... I have noticed earlier today that it is also unused.